### PR TITLE
Fixed compilation + refactored.

### PR DESCRIPTION
**Pull request by nazarii-gnydyn**
Not for merge.

### CODE REVIEW

File: test/saithrift/src/switch_sai.thrift [hunk 1]
```diff
@@ -191,70 +199,75 @@ struct sai_thrift_attribute_list_t {
...
     sai_thrift_object_id_t sai_thrift_create_next_hop(1: list<sai_thrift_attribute_t> thrift_attr_list);
     sai_thrift_status_t sai_thrift_remove_next_hop(1: sai_thrift_object_id_t next_hop_id);
 
-    //next hop group API
+    // Next Hop Group API.
     sai_thrift_object_id_t sai_thrift_create_next_hop_group(1: list<sai_thrift_attribute_t> thrift_attr_list);
     sai_thrift_status_t sai_thrift_remove_next_hop_group(1: sai_thrift_object_id_t next_hop_group_id);
-    sai_thrift_status_t sai_thrift_add_next_hop_to_group(1: sai_thrift_object_id_t next_hop_group_id, 2: list<sai_thrift_object_id_t> thrift_nexthops);
-    sai_thrift_status_t sai_thrift_remove_next_hop_from_group(1: sai_thrift_object_id_t next_hop_group_id, 2: list<sai_thrift_object_id_t> thrift_nexthops);
+    sai_thrift_result_t sai_thrift_add_next_hop_to_group(1: list<sai_thrift_attribute_t> thrift_attr_list);
```
Comment (MaxPolovyi) [inline test/saithrift/src/switch_sai.thrift:247]: I suggest you to rename this accordingly to SAI function name (sai_thrift_create_next_hop_group_member)

File: test/saithrift/src/switch_sai.thrift [hunk 2]
```diff
@@ -191,70 +199,75 @@ struct sai_thrift_attribute_list_t {
...
+    sai_thrift_status_t sai_thrift_set_stp_port_state(1: sai_thrift_object_id_t stp_id,
+                                                      2: sai_thrift_object_id_t port_id,
+                                                      3: sai_thrift_port_stp_port_state_t stp_port_state);
+    sai_thrift_port_stp_port_state_t sai_thrift_get_stp_port_state(1: sai_thrift_object_id_t stp_id,
+                                                                   2: sai_thrift_object_id_t port_id);
+
+    // Neighbor API.
+    sai_thrift_status_t sai_thrift_create_neighbor_entry(1: sai_thrift_neighbor_entry_t thrift_neighbor_entry,
+                                                         2: list<sai_thrift_attribute_t> thrift_attr_list);
     sai_thrift_status_t sai_thrift_remove_neighbor_entry(1: sai_thrift_neighbor_entry_t thrift_neighbor_entry);
```
Comment (MaxPolovyi) [inline test/saithrift/src/switch_sai.thrift]: rename to sai_thrift_delete_neighbor_entry

**Review by MaxPolovyi — COMMENTED** [review]
in SAI the '...delete...' functions are now called '...remove...'. Please check if you've renamed all of thrift APIs accordingly, and rename what's left.

Also, I believe it'd be much easier to review, if the 'refactoring' was in separate pull request. Especially such minor things like comments' case changes.

**Comment by nazarii-gnydyn** [comment]
959a5b0: Also fixed ACL test.

**Review by ghost — APPROVED** [review]
Looks good to me

**Pull request closed without merging**